src/app/config.cc

```diff
@@ -167,6 +173,13 @@ void Configuration::ParseKovriConfigFile(
   bpo::notify(var_map);
 }
 
+void Configuration::SetupGlobalPath() {
```

Comment:
Did your version of clang-format not want to newline break this brace? If so, we should resolve any versioning issue like we did on IRC with cpplint.